arm/Microsoft.KeyVault/vaults/secrets/readme.md

```diff
@@ -14,7 +14,7 @@ This module deploys a key vault secret.
 | Parameter Name | Type | Default Value | Possible Values | Description |
 | :-- | :-- | :-- | :-- | :-- |
 | `attributesEnabled` | bool | `True` |  | Optional. Determines whether the object is enabled. |
-| `attributesExp` | int | `-1` |  | Optional. Expiry date in seconds since 1970-01-01T00:00:00Z. |
+| `attributesExp` | int | `-1` |  | Optional. Expiry date in seconds since 1970-01-01T00:00:00Z. Security recommendation is to set expiration date whenever possible. |
```

Comment:
```suggestion
| `attributesExp` | int | `-1` |  | Optional. Expiry date in seconds since 1970-01-01T00:00:00Z. For security reasons, it is recommended to set an expiration date whenever possible. |
```

Reply:
Update description in template too